README.md

```diff
@@ -0,0 +1,68 @@
+ngrok-nix
+=========
+
+Run ngrok as a service on your NixOS machine
+
+configuration.nix
+---------
+
+When using `configuration.nix` you can include ngrok service as follows:
+
+```nix
+{ pkgs, config, ... }:
+let
+  ngrok = builtins.fetchGit {
+    url = "https://github.com/ngrok/ngrok-nix";
+    rev = "c56189898f263153a2a16775ea2b871084a4efb0";
+  };
+in
+{
+  nixpkgs.config.allowUnfree = true;
+  imports = [
+    "${ngrok}/nixos.nix"
+  ];
+  services.ngrok = {
+    enable = true;
+    extraConfig = {
+      authtoken = "<YOUR_AUTHTOKEN>";
+    };
```

Comment:
Let's not include this in our example - it's not great practice to include secrets directly in your nixos config. We should point people to the extra config _files_ option instead.

Reply:
oh, yeah, I was planning to expand on this with pulling the secrets from somewhere else 👍